virocon/plotting.py

```diff
@@ -60,6 +125,23 @@ def ppf(self, q):
     return axes
 
 def plot_dependence_functions(model, model_desc=None, par_rename=None, axes=None):
+    """
+    Function to plot the dependence functions of the indicated model.
+       
+    Parameters
+    ----------
+    model :  MultivariateModel
+        The model to be used to plot the dependence functions.      
+    model_desc: dict, optional
+        Generated model description. Defaults to None.
+    par_rename : optional
+        ?
+    axes: int, optional
+        Indicates the number of subplots. If not further specified, axes are 
```

Comment:
list, optional
The matplotlib axes objects to plot into. One for each conditional parameter. If None (the default) a new figure will be created for each conditional parameter.